### ISSUE TITLE
Remapping alt-\ to open the file in vertical split throws error

### ISSUE THREAD

**Issue by rbhanot4739** [scrooloose/nerdtree#947]
### Environment

* Operating System: Ubuntu 16.04.5 LTS
* Vim version `:NVIM v0.3.2-208-g2841e89`:  
* NERDTree version `5.0.0`: 
* NERDTree settings applied in your vimrc, if any:
    ```vim
    ```

### Process

1. Define these custom key bindings 
```
let g:NERDTreeMapOpenVSplit = '<a-\>'
```

### Current Result

So inside the NERDTree window when I press `a-\` I get the following error.
```
E114: Missing quote: "\")                                                                                                                                        
E116: Invalid arguments for function nerdtree#ui_glue#invokeKeyMap 
```
### Expected Result

I have mapped same key-stroke in normal for creating vsplits and it works fine. So what am i missing here ?

**Comment by PhilRunninger** [comment]
Fixed in #948.

**Comment by rbhanot4739** [comment]
Thank you very much, works like charm !!
